app/views/users/new.html.haml

```diff
@@ -0,0 +1,24 @@
+%h1 Sign Up
+#new-form
+= form_for(@user) do |f|
```

Comment:
This line and all after should get indented to nest the form inside the new-form div